src/pages/Editor.vue

```diff
@@ -6,10 +6,10 @@
       :museum-info="HeaderData.museumInfo"
       :has-search="HeaderData.hasSearch"
       :force-on-dark="HeaderData.forceOnDark"
-      :sticky="true"
     />
     <Hero name="Content" :data="heroData" />
-    <section>
+    <section class="content">
+      <RichText :content="introText" contentType="json" />
```

Comment:
would it be easier to use the Markdown variant of this for enabling future edits?

Reply:
@abrin true, it would - I can change it on the next pass?